pkg/process/checks/process.go

```diff
@@ -99,6 +104,31 @@ func (p *ProcessCheck) Init(_ *SysProbeConfig, info *HostInfo) error {
 	initScrubber(p.scrubber)
 
 	p.disallowList = initDisallowList()
+
+	p.lastConnRates = atomic.NewPointer[ProcessConnRates](nil)
+	p.connRatesReceiver = subscriptions.NewReceiver[ProcessConnRates]()
+
+	if p.connRatesReceiver.Ch != nil {
```

Comment:
Would it be possible to test the pub/sub behavior? The logic isn't complex per-se and I trust that it works, but it is somewhat new and I just want to make sure that we have coverage for this